components/molecule/tabs/src/index.js

```diff
@@ -21,7 +21,9 @@ class MoleculeTabs extends Component {
   _createHandleChange(index) {
     return event => {
       event.preventDefault()
-      if (this.props.items[index].enabled !== false) {
+      const {items, allowHandleDisabledTabClick} = this.props
+      const {enabled} = items[index]
+      if (enabled !== false || allowHandleDisabledTabClick) {
```

Comment:
Instead of `enabled !== false` we could: `enabled === true` that is the same than just `enabled`, right? Or am I missing something?  And yes, it should be `&&` good catch @miduga

Reply:
isn't the same. enabled can be any value, "" 1 "hello" etc. with enabled === true we don't cover those cases

Reply:
`!!enabled === true` ?
:trollface:

Reply:
we should avoid blocking PR because this kind of stuff, it works as it is, it has no performance boost and everyone can read it even if it is the negative way around